backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -125,6 +125,14 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
       webName = "text",
       componentsToReplace = setOf("androidx.compose.material.Text")
     ),
+    TEXT_FIELD(
+      fullName = "net.skyscanner.backpack.compose.textfield.BpkTextField",
+      webName = "text-input",
+      componentsToReplace = setOf(
+        "androidx.compose.foundation.text.BasicTextField",
```

Comment:
probably also good to add the other variants here - particularly outline as that is very similar to our component